config/server-vars.yml

```diff
@@ -114,21 +111,69 @@ COMPREHENSIVE_THEME_DIRS: "{{ EDXAPP_COMPREHENSIVE_THEME_DIRS }}"
 EDXAPP_GRADE_STORAGE_KWARGS:
   container: "reports"
 EDXAPP_GRADE_STORAGE_CONTAINER: "reports"
+
 # overriding the default value from 5 to 50. we don't want to stop the retry's at any point.  
 EDXAPP_RETRY_ACTIVATION_EMAIL_MAX_ATTEMPTS: 50
+
 # overriding the default value from 0.5 to 2. 
 EDXAPP_RETRY_ACTIVATION_EMAIL_TIMEOUT: 2
+
+# Security
 EDXAPP_SESSION_INACTIVITY_TIMEOUT_IN_SECONDS: 1800
 EDXAPP_X_FRAME_OPTIONS: "SAMEORIGIN"
+NGINX_ENABLE_SSL: %%NGINX_ENABLE_SSL%%
+NGINX_REDIRECT_TO_HTTPS: %%NGINX_ENABLE_SSL%%
 EDXAPP_CSRF_COOKIE_SECURE: %%NGINX_ENABLE_SSL%%
 
 EDXAPP_LTI_USER_EMAIL_DOMAIN: "lti.openedx.microsoft.com"
 
 # Default log levels: DEBUG, INFO, WARNING, ERROR, CRITICAL
 #EDXAPP_LOG_LEVEL: 'INFO'
 
-#Cookies API url
+# Cookies API url
 EDXAPP_API_COOKIE_URL: "%%EDXAPP_COOKIE_API_URL%%"
 
 # View live drop down for CMS
 EDXAPP_ENABLE_MULTI_SITE_VIEWLIVE_DROPDOWN: "%%EDXAPP_ENABLE_MULTI_SITE_VIEWLIVE_DROPDOWN%%"
+
+# Shared Mongo Settings
+MONGO_BIND_IP: "0.0.0.0"
+EDXAPP_MONGO_USE_SSL: False
+EDXAPP_MONGO_USER: '%%MONGO_USER%%'
+EDXAPP_MONGO_PASSWORD: '%%MONGO_PASSWORD%%'
+
+# Shared Forum Settings
+FORUM_MONGO_DATABASE: "cs_comments_service"
+FORUM_MONGO_PORT: "27017"
+FORUM_MONGO_TAGS: !!null
+FORUM_MONGO_USER: '%%MONGO_USER%%'
+FORUM_MONGO_PASSWORD: '%%MONGO_PASSWORD%%'
+FORUM_MONGO_URL: "mongodb://{{ FORUM_MONGO_USER }}:{{ FORUM_MONGO_PASSWORD }}@{%- for host in FORUM_MONGO_HOSTS -%}{{ host }}:{{ FORUM_MONGO_PORT }}{%- if not loop.last -%},{%- endif -%}{%- endfor -%}/{{ FORUM_MONGO_DATABASE }}?replicaSet={{ MONGO_REPLICASET_NAME }}{%- if FORUM_MONGO_TAGS -%}&tags={{ FORUM_MONGO_TAGS }}{%- endif -%}"
+
+# Shared mysql settings
+EDXAPP_MYSQL_USER_ADMIN: '%%MYSQL_ADMIN_USER%%'
+EDXAPP_MYSQL_PASSWORD_ADMIN: '%%MYSQL_ADMIN_PASSWORD%%'
+EDXAPP_MYSQL_USER: '%%MYSQL_USER%%'
+EDXAPP_MYSQL_PASSWORD: '%%MYSQL_PASSWORD%%'
+COMMON_MYSQL_MIGRATE_USER: "{{ EDXAPP_MYSQL_USER }}"
```

Comment:
@manikarthikk , the MYSQL_MIGRATE keys were missing from onebox. Moving it to server-vars means that onebox deployments should get these values now. I believe this solves the bug you were encountering, but we should eventually verify this.